service/rtc/session.go

```diff
@@ -77,9 +77,9 @@ type session struct {
 	videoRateMonitors map[string]*RateMonitor
 
 	// Receiver
-	bwEstimator       cc.BandwidthEstimator
-	screenTrackSender *webrtc.RTPSender
-	rxTracks          map[string]webrtc.TrackLocal
+	bwEstimator        cc.BandwidthEstimator
+	screenTrackSenders map[string]*webrtc.RTPSender
```

Comment:
Now we have to support potentially concurrent screen tracks, one per encoding (VP8 and AV1 at the moment).